tests/phpunit/admin/class-wc-rest-stripe-agentic-commerce-controller-test.php

```
@@ -0,0 +1,566 @@
+		$response = rest_do_request( $request );
+
+		$this->assertEquals( 200, $response->get_status() );
+
+		$data = $response->get_data();
+		$this->assertArrayHasKey( 'last_sync', $data );
+		$this->assertArrayHasKey( 'history', $data );
+		$this->assertArrayHasKey( 'next_sync', $data );
+		$this->assertNull( $data['last_sync'] );
+		$this->assertSame( [], $data['history'] );
+		$this->assertNull( $data['next_sync'] );
+	}
```

PHPMD (2.15.0)

[error] The method test_get_status_returns_empty_state is not named in camelCase. (undefined)

(CamelCaseMethodName)